CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/Memory.java

```diff
@@ -34,4 +41,58 @@ public void flushMemory(){
         _memory.clear();
         _address = 0;
     }
+
+    public String copyFiles(Args args) throws IOException {
+        File sourcePath = new File(args.get("source_path").toString());
+        File destinationPath = new File(args.get("destination_path").toString());
+        try {
+            copyFolder(sourcePath, destinationPath);
```

Comment:
is it possible to get status of copyFolder whether it is copied successfully or not.

Reply:
this a method which recursively copies files and folder, so if copy is successful, you won't get any issue, other wise and exception would be thrown. Sending an explicit string doesn't make any sense